2019-11-27,18:08:08 [DEBUG] (app.py:app:215): Parameter port set to "9004" from default value
2019-11-27,18:08:08 [DEBUG] (app.py:app:215): Parameter server_version set to "0.8.9" from default value
2019-11-27,18:08:08 [DEBUG] (app.py:app:215): Parameter evaluate_timeout set to "30" from default value
2019-11-27,18:08:08 [DEBUG] (app.py:app:215): Parameter upload_dir set to "/Users/dae/anaconda3/lib/python3.6/site-packages/tabpy/tmp/query_objects" from default value
2019-11-27,18:08:08 [DEBUG] (app.py:app:215): Parameter transfer_protocol set to "http" from default value
2019-11-27,18:08:08 [DEBUG] (app.py:app:221): Parameter certificate_file is not set
2019-11-27,18:08:08 [DEBUG] (app.py:app:221): Parameter key_file is not set
2019-11-27,18:08:08 [DEBUG] (app.py:app:215): Parameter state_file_path set to "/Users/dae/anaconda3/lib/python3.6/site-packages/tabpy/tabpy_server" from default value
2019-11-27,18:08:08 [INFO] (app.py:app:280): Loading state from state file /Users/dae/anaconda3/lib/python3.6/site-packages/tabpy/tabpy_server/state.ini
2019-11-27,18:08:08 [DEBUG] (app.py:app:215): Parameter static_path set to "./" from default value
2019-11-27,18:08:08 [DEBUG] (app.py:app:292): Static pages folder set to "/Users/dae/Documents/github_0day/ga/projects/personal_git/nyc_homeless_pop_prediction"
2019-11-27,18:08:08 [DEBUG] (app.py:app:221): Parameter TABPY_PWD_FILE is not set
2019-11-27,18:08:08 [INFO] (app.py:app:311): Password file is not specified: Authentication is not enabled
2019-11-27,18:08:08 [DEBUG] (app.py:app:215): Parameter log_request_context set to "false" from default value
2019-11-27,18:08:08 [INFO] (app.py:app:327): Call context logging is disabled
2019-11-27,18:08:08 [DEBUG] (app.py:app:215): Parameter max_request_size_in_mb set to "100" from default value
2019-11-27,18:08:08 [INFO] (app.py:app:110): Initializing TabPy...
2019-11-27,18:08:08 [INFO] (callbacks.py:callbacks:42): Initializing TabPy Server...
2019-11-27,18:08:08 [DEBUG] (state.py:state:147): Collected endpoints: {}
2019-11-27,18:08:08 [INFO] (app.py:app:113): Done initializing TabPy.
2019-11-27,18:08:08 [INFO] (app.py:app:67): Setting max request size to 104857600 bytes
2019-11-27,18:08:08 [INFO] (callbacks.py:callbacks:62): Initializing models...
2019-11-27,18:08:08 [DEBUG] (state.py:state:147): Collected endpoints: {}
